lib/logstash/outputs/http.rb

```diff
@@ -128,6 +136,45 @@ def run
       @pending << [@event, @attempt]
     end
   end
+
+  def send_json_batch(events)
+    attempt = 1
+    body = LogStash::Json.dump(events.map {|e| map_event(e) })
+    begin
+      while true
+        request = client.send(@http_method, @url, :body => body, :headers => @headers)
+        response = request.call
+        break if response_success?(response)
+        if retryable_response?(response)
+          handle_retryable_response(response)
```

Comment:
`handle_retryable_response` isn't implemented, resulting in:

```
[2018-01-05T23:19:32,141][ERROR][logstash.javapipeline    ] Exception in pipelineworker, the pipeline stopped processing new events, please check your filter configuration and restart Logstash. {:pipeline_id=>"main", "exception"=>"undefined method `handle_retryable_response' for #<LogStash::Outputs::Http:0x43b294b6>", "backtrace"=>["/Users/joaoduarte/projects/logstash_plugins/logstash-output-http/lib/logstash/outputs/http.rb:149:in `send_json_batch'",
```

Reply:
Ah, that was supposed to call `log_retryable_response`